packages/jest-worker/src/__tests__/PriorityQueue.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import PriorityQueue from '../PriorityQueue';
+import {CHILD_MESSAGE_CALL} from '../types';
+
+it('returns the tasks in order', () => {
+  const computePriority = (_method, task) => task.priority;
+  const queue = new PriorityQueue(computePriority);
+  const priorities = [10, 3, 4, 8, 2, 9, 7, 1, 2, 6, 5];
+
+  for (const priority of priorities) {
+    queue.enqueue(createQueueChildMessage({priority}));
+  }
+
+  priorities.sort((a, b) => a - b);
```

Comment:
```suggestion
  priorities.sort();
```

Reply:
The override is required because JavaScript does by default a lexicographical sort. 

> The sort() method sorts the elements of an array in place and returns the sorted array. The default sort order is ascending, built upon converting the elements into strings, then comparing their sequences of UTF-16 code units values. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)


```
[10, 3, 4, 8, 2, 9, 7, 1, 2, 6, 5].sort()
Array(11) [ 1, 10, 2, 2, 3, 4, 5, 6, 7, 8, … ]
```

Reply:
JS strikes again 😀